<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeedCalculator</class>
 <widget class="QWidget" name="SeedCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seed Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxResults">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string notr="true">%p%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="progressLabel"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidgetStars">
     <widget class="QWidget" name="tab35">
      <attribute name="title">
       <string>3-5★</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="RaidInfo35" name="raidInfo35" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab12">
      <attribute name="title">
       <string>1-2★</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="RaidInfo12" name="raidInfo12" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSearchSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxDen"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRarity">
        <property name="text">
         <string>Rarity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelIVDeviation">
          <property name="text">
           <string>Search Range</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxIVDeviationMin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxIVDeviationMax">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxRarity">
        <item>
         <property name="text">
          <string>Common</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rare</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDen">
        <property name="text">
         <string>Den</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxGame">
        <item>
         <property name="text">
          <string>Sword</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shield</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxStop">
        <property name="text">
         <string>Stop after first result</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxDay6">
        <property name="text">
         <string>Use day 6</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RaidInfo35</class>
   <extends>QWidget</extends>
   <header>Forms/Util/RaidInfo35.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RaidInfo12</class>
   <extends>QWidget</extends>
   <header>Forms/Util/RaidInfo12.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxDen</tabstop>
  <tabstop>comboBoxRarity</tabstop>
  <tabstop>comboBoxGame</tabstop>
  <tabstop>spinBoxIVDeviationMin</tabstop>
  <tabstop>spinBoxIVDeviationMax</tabstop>
  <tabstop>checkBoxStop</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>tabWidgetStars</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
